lib/order.rb

```diff
@@ -0,0 +1,84 @@
+require 'twilio-ruby'
+require_relative '../.env.rb'
+
+class Order 
+
+  attr_accessor :basket
+  attr_reader :total, :menu
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @basket = {}
+    @total = 0.0
+  end
+
+  def check_dishes
+    list_of_dishes = ''
+    menu.dishes_available.each do |dish, price|
+      list_of_dishes += ("#{dish} £#{price}\n")
+    end
+    list_of_dishes
+  end
+
+  def add_item(dish, quantity = 1)
+    fail 'That item is not on the menu' unless menu.dishes_available.has_key?(dish)
```

Comment:
Metrics/LineLength: Line is too long. [83/80]<br>Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.